source/managing_security/users.rst

```diff
@@ -74,28 +94,20 @@ Other permissions
    * * Permission
      * Description
    * * Edit domain properties
-     * The user may edit all properties in the **Look & feel**, **Users & groups** and **Configuration** sections of
-       the backoffice. He/she also gets access to the **Analytics** and **Assets** sections.
-
-       **Grants access to the catalog section of the backoffice**
-   * * Edit all pages
-     * The user may create, edit and delete all pages. He/she can also manage assets.
-
-       **Grants access to the catalog section of the backoffice**
+     * The user may configure all options and parameters in the Assets, Look & feel, Users & groups and Configuration sections of the back office.
+       **This permission grants access to the Assets, Look & feel, Users & groups, Configuration and License sections of the back office.**
```

Comment:
Not true, the Theme section is under the Look & Feel section but require the edit_theme property